### DISCUSSION TITLE
Acceptable Diodes

### DISCUSSION THREAD

**Discussion by bland-industries** [strayer/taira-keyboard#2] [Q&A] [answered]
I am not an electrical engineer, so some of this is over my head. I am trying to find and purchase acceptable diodes to use on my Tiara build. I am not finding great results when I search "SOD-123FL".

So I was wondering if this product was an acceptable diode to use on my build: https://www.adafruit.com/product/5099.

Thank you for any help you can provide.

**Comment by strayer** [comment, marked as answer]
Hey,

I'm pretty sure I bought these: https://keycapsss.com/keyboard-parts/parts/138/smd-diodes-sod-123fl?number=KC10103

Based on the datasheet they are 1.1mm (SOD-123FL) thick compared to 1.35mm (SOD-123). They are also a bit shorter. But then again, the Kailh hotswap sockets are at least 1.8mm thick and take the most space between the PCB and bottom plate, so it shouldn't be a problem to use SOD-123 diodes.

Please let me know if they fit! This should definitely be mentioned in the build guide.
